analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/RuleChecks/EmptyCollectionsShouldNotBeEnumeratedBase.cs

```diff
@@ -200,6 +215,21 @@ private static ProgramState ProcessAddMethod(ProgramState state, IMethodSymbol m
             {
                 state = state.SetSymbolConstraint(symbol, CollectionConstraint.NotEmpty);
             }
+            return state;
+        }
+        return null;
+    }
+
+    private static ProgramState ProcessRemoveMethod(ProgramState state, IMethodSymbol method, IOperation instance)
+    {
+        if (RemoveMethods.Contains(method.Name))
+        {
+            var value = (state[instance] ?? SymbolicValue.Empty).WithoutConstraint(CollectionConstraint.NotEmpty);
+            state = state.SetOperationValue(instance, value);
+            if (instance.TrackedSymbol(state) is { } symbol)
```

Comment:
For some reason, coverage finds two conditions here (of which only one is covered). I only see a null-check, so not sure how to increase coverage.

Reply:
I guess you only have a test that covers the symbol is not null case.
Is there a case where this could be null?